### ISSUE TITLE
How to Contribute to DP-600 Training Modules

### ISSUE THREAD

**Issue by JamesDBartlett3** [microsoft/Fabric_Workshops#28]
Hello! 

I've been working my way through all of [the modules in the DP-600 training collection](https://learn.microsoft.com/en-us/credentials/certifications/exams/dp-600/) recently, and I've noticed a fair amount of inaccurate/outdated material. 

Example: The [Understand calculation groups](https://learn.microsoft.com/en-us/training/modules/create-calculation-groups/2-understand) training module claims that "Power BI Desktop doesn't have the user interface to create calculation groups."
![image](https://github.com/microsoft/Fabric_Workshops/assets/37491308/26885fc1-c54a-4b86-878c-9323791ad34d)
However, [Power BI Desktop gained native support for creating calculation groups in October 2023](https://learn.microsoft.com/en-us/power-bi/transform-model/calculation-groups) with the introduction of the [Model explorer view](https://learn.microsoft.com/en-us/power-bi/transform-model/model-explorer), so that claim is now inaccurate/outdated.

I've contributed to several other Microsoft docs repositories in the past, so when I discovered these issues, I started looking for the source code in hopes that I could fork the repo, fix the issues, and submit a PR. Unfortunately, the closest thing I could find is this repository, which [contains links to the training modules in question](https://github.com/microsoft/Fabric_Workshops/tree/main/docs/04-microsoft-learn-resources), but doesn't contain their source code.

_**TL;DR: I'd like to help correct the inaccuracies in these training modules, but I can't find the source code for them. Can someone point me in the right direction?**_

Thank you!

James

**Comment by AnthonyByansi** [comment]
Hi @JamesDBartlett3 
Referencing the same [documentation](https://learn.microsoft.com/en-us/power-bi/transform-model/calculation-groups?WT.mc_id=%3Fwt.mc_id%3Dstudentamb_260352) source, there is a pencil icon up above the documentation, please proceed to log the issue on the PowerBI documentation Repository
![Contribute](https://github.com/microsoft/Fabric_Workshops/assets/101401469/b378da42-045c-498c-8d7b-9d90bbacf6b9)


![image](https://github.com/microsoft/Fabric_Workshops/assets/101401469/452b2f3d-5f8c-4ea2-935b-d4f02bf5a20c)

**Comment by JamesDBartlett3** [comment]
> Hi @JamesDBartlett3 Referencing the same [documentation](https://learn.microsoft.com/en-us/power-bi/transform-model/calculation-groups?WT.mc_id=%3Fwt.mc_id%3Dstudentamb_260352) source, there is a pencil icon up above the documentation

Hi @AnthonyByansi, [the link you provided](https://learn.microsoft.com/en-us/power-bi/transform-model/calculation-groups?WT.mc_id=%3Fwt.mc_id%3Dstudentamb_260352) points to an MS Learn **_documentation article_**, whereas the inaccuracies that I pointed out in my comment above are [in an MS Learn **_training module_**](https://learn.microsoft.com/en-us/training/modules/create-calculation-groups/2-understand), which **_does not_** feature a pencil icon at the top of the page. This is why I opened this issue, because there does not appear to be a way for me to contribute edits to MS Learn **_training modules_** in the same way that I can contribute edits to MS Learn **_documentation articles_**.
